lib/message.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'database_connection'
+
+class Message
+
+  attr_reader :id, :content, :time, :user_name, :user_handle
+
+  def initialize(id:, content:, time:, user_name:, user_handle:)
```

Comment:
I like the use of named parameters here.  IT's generally a good idea for any method with a long list of arguments.